xonsh/completer.py

```diff
@@ -136,10 +138,19 @@ def cmd_complete(self, cmd):
         return {s + space for s in self._all_commands() if s.startswith(cmd)}
 
     def module_complete(self, prefix):
-        """Completes a name of a module to import"""
+        """Completes a name of a module to import."""
         modules = set(sys.modules.keys())
         return {s for s in modules if s.startswith(prefix)}
 
+    def option_complete(self, prefix, cmd):
+        """Completes an option name, basing on content of man page."""
+        manpage = subprocess.Popen(["man", cmd], stdout=subprocess.PIPE)
+        # This is a trick to get rid of reverse line feeds
+        text = subprocess.check_output(["col", "-b"], stdin=manpage.stdout).decode('utf-8')
```

Comment:
Ahhh. Ok.  I hadn't considered the Mac OSX issue, but you are right.  What I was planning on doing originally was using PLY to write a simple lexer and parser for the troff language.  I have a 'man' branch on my repo that started this if you want to check it out...